test/Npgsql.Tests/Types/MiscTypeTests.cs

```diff
@@ -11,6 +12,9 @@ namespace Npgsql.Tests.Types;
 /// </summary>
 class MiscTypeTests : MultiplexingTestBase
 {
+    NpgsqlDataSource _slimWithoutMappingsDataSource = null!;
```

Comment:
Not sure there's any point to have these on the class - can just have two tests which each manages its own builder inside the tests.

See e.g. [what I did for ranges](https://github.com/npgsql/npgsql/pull/4989/files#diff-2c28aca7be08f5f3fef4d5ee05a42c86c4acdb44726cb618cdac19a171c7831bR252).

Reply:
> Not sure there's any point to have these on the class - can just have two tests which each manages its own builder inside the tests.

The point was that we're going to split BuiltInTypeResolver in quite a few resolvers. so we can reuse the exact same data source between the tests for each type.

Reply:
As you wish - though I'm not sure it's worth doing that kind of stuff (with class setup/teardown)... Can just as well have every test completely self-contained, adding (or not) exactly what they want on the slim builder

Reply:
You have my eternal gratitude :wink: